src/library/simplified/tag/criteo.js

```diff
@@ -0,0 +1,472 @@
+//~~tv:20010.20140827
```

Comment:
why I cannot see any diff? Was it because that file was never pushed to github originally?